content/intro-to-storybook/react/es/using-addons.md

```diff
@@ -0,0 +1,129 @@
+
+Ahora solucionemos el problema con el desbordamiento agregando un estilo a `Task.js`:
+
+```diff:title=src/components/Task.js
+export default function Task({ task: { id, title, state }, onArchiveTask, onPinTask }) {
+  return (
+    <div className={`list-item ${state}`}>
+      <label
+        htmlFor="checked"
+        aria-label={`archiveTask-${id}`}
+        className="checkbox"
+      >
+        <input
+          type="checkbox"
+          disabled={true}
+          name="checked"
+          id={`archiveTask-${id}`}
+          checked={state === "TASK_ARCHIVED"}
+        />
+        <span
+          className="checkbox-custom"
+          onClick={() => onArchiveTask(id)}
+        />
+      </label>
+
+      <label htmlFor="title" aria-label={title} className="title">
+        <input
+          type="text"
+          value={title}
+          readOnly={true}
+          name="title"
+          placeholder="Input title"
++         style={{ textOverflow: 'ellipsis' }}
+        />
+      </label>
+
+      {state !== "TASK_ARCHIVED" && (
+        <button
+          className="pin-button"
+          onClick={() => onPinTask(id)}
+          id={`pinTask-${id}`}
+          aria-label={`pinTask-${id}`}
+          key={`pinTask-${id}`}
+        >
+          <span className={`icon-star`} />
+        </button>
+      )}
+    </div>
+  );
+}
+```
+
+![That's better.](/intro-to-storybook/edge-case-solved-with-controls-6-4.png)
+
+Hemos resuelto el problema. Cuando el texto alcanza el límite del área de Task, usamos unos puntos suspensivos para truncar el texto.
+
+### Agregando una nueva historia para evitar regresiones
+
+En el futuro, podemos reproducir manualmente este problema ingresando la misma cadena a través de Controls. Pero es más fácil escribir una historia que muestre este caso extremo. Eso amplia nuestra cobertura de pruebas de regresión y describe claramente los límites de los componentes para el resto del equipo.
+
+Agrega una nueva historia para el caso de texto largo en `Task.stories.js`:
+
+```js:title=src/components/Task.stories.js
+const longTitleString = `This task's name is absurdly large. In fact, I think if I keep going I might end up with content overflow. What will happen? The star that represents a pinned task could have text overlapping. The text could cut-off abruptly when it reaches the star. I hope not!`;
+
+export const LongTitle = Template.bind({});
+LongTitle.args = {
+  task: {
+    ...Default.args.task,
+    title: longTitleString,
+  },
+};
+```
+
+Ahora podemos reproducir y trabajar en este caso extremo con facilidad.
+
+<video autoPlay muted playsInline loop>
+  <source
+    src="/intro-to-storybook/task-stories-long-title-6-4.mp4"
+    type="video/mp4"
+  />
+</video>
+
+Si estamos haciendo [pruebas visuales](/intro-to-storybook/react/es/test/), las pruebas nos dirán si la solución de truncamiento se rompe. Los casos extremos oscuros pueden olvidarse sin la cobertura de prueba.
```

Comment:
@lmac-1, when you're able, could you update this sentence to:
```md
Si estamos haciendo [pruebas visuales] (/intro-to-storybook/react/es/test/), las pruebas nos dirán si la solución de truncamiento se rompe. Los casos extremos podrían ser ignorados sin la cobertura de prueba.
```
Per my conversation with my fellow team member, that is way more fluent than me in Spanish, the choice of `oscuros` is not incorrect not may have the wrong connotation. She also pointed out that the original English version should include the same change.

Reply:
Fixed in commit 665ee7c 🙏🏼